bundles/org.openhab.binding.govee/src/main/java/org/openhab/binding/govee/internal/CommunicationManager.java

```diff
@@ -224,7 +225,9 @@ public void run() {
                                     }
                                 }
                             } catch (JsonParseException e) {
-                                // this probably was a status message
+                                logger.debug(
+                                        "JsonParseException when trying to parse the response, probably a status message");
```

Comment:
It might be useful to log the exception details as well:
```suggestion
                                        "JsonParseException when trying to parse the response, probably a status message, e");
```

Reply:
You mean

```
"JsonParseException when trying to parse the response, probably a status message", e); 
```

;-)

Reply:
Yes. 😄

Reply:
Fixed.